source/percona-release.rst

```diff
@@ -356,26 +356,19 @@ Example: All steps for installing a specific Percona product
 Updating |percona-release| to the Latest Version
 ================================================================================
 
-|percona-release| itself is available from the *original* repository. Thus, the
-update procedure is very simple:
+|percona-release| itself is available from the *prel* repository which is enabled by default. Thus, to update |percona-release|, simply run the following command:
 
-#. Enable the *original* repository
+- For RPM-based systems:
 
-   .. code-block:: bash
-
-      $ sudo percona-release enable original
+  .. code-block:: bash
 
-#. Install the latest version using the package manager of your system:
+     $ yum update percona-release
 
-   .. code-block:: bash
+- For DEB-based systems:
 
-      # RPM-based systems
-      $ yum update percona-release
-
-   .. code-block:: bash
+  .. code-block:: bash
 
-      #DEB-based systems
-      $ apt-get install percona-release
+     $ apt-get install percona-release
```

Comment:
maybe we also need to add `apt-get update` command as apt database need to be updated in order to understand are there any updates or not